qemu/tests/cgroup.py

```diff
@@ -306,8 +307,8 @@ def _test(direction):
                     err += "%d, " % i
 
             logging.info("blkio_bandwidth_%s: dd statistics\n%s", direction,
-                         utils.matrix_to_string(out, ['status', 'norm_weights',
-                                                      'norm_out', 'actual']))
+                         process.matrix_to_string(out, ['status', 'norm_weights',
```

Comment:
just noticed that API matrix_to_string() is not available in avocado, possible solution is
1. Earlier, you have used utils.matrix_to_string() but utils was not imported led to error - so you can use utils.matrix_to_string() by importing utils.
2. If the API can be used by many test in future, can be added to avocado-vt and then used in your test.